src/redux/store.jsx

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookSlice from './books/bookSlice';
+import categoriesSlice from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    books: bookSlice.reducer,
+    categories: categoriesSlice.reducer,
```

Comment:
- You have done great so far implementing most of the changes suggested by the previous reviewer. :+1: :+1: But there are some requirements you still need to meet before moving to the next milestone. :+1: 

- [ ] Looking at the video below, you can see that your project gives an error when starting the server and that prevents the react app from running, I'll suggest you look into this as your bookstore app is meant to be running correctly. :+1: 

- - Video reference:

https://github.com/Zven94/Bookstore/assets/107632780/890e48e1-0240-4fb6-932a-7c6d91fb1ee9

- [ ] Also I'll suggest you use the `.js` extension for your redux files instead of `.jsx`.  JSX is an addition to the JavaScript syntax which is a mixture of both HTML and JavaScript and the redux files don't have HTML. :wave: :+1:

Reply:
Hi! Thank you for your time! I found the issue.
In the line 13 on the file: index.js i change
`import store from './redux/store.js';`
for:
`import store from './redux/store';`
and now it's running

Reply:
now i'm having this issue, is it normal?:
![image](https://github.com/Zven94/Bookstore/assets/125831572/b65e3689-6362-468f-9570-5fe9b6ea9180)